src/calendar/calendar.style.tsx

```diff
@@ -0,0 +1,159 @@
+import { Color } from "../color";
+import { ChevronLeftIcon } from "@lifesg/react-icons/chevron-left";
+import { ChevronDownIcon } from "@lifesg/react-icons/chevron-down";
+import styled, { css } from "styled-components";
+import { TextStyleHelper } from "../text/helper";
+import { View } from "./calendar";
+import { Text } from "../text";
+import { CalendarProps } from "./types";
+import { MonthPickerContainer } from "./calendar-month.style";
+import { YearPickerContainer } from "./calendar-year.style";
+
+interface ContainerProps extends Pick<CalendarProps, "$type"> {}
+
+interface HeaderDropdownProps {
+    $view: View;
+}
+
+interface CalendarViewProps {
+    children: JSX.Element[];
```

Comment:
why do you need this prop?

Reply:
I will removed this `children` prop, the typescript not complaint anymore. :'))